dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -17,6 +18,7 @@
     private Class<?> javaType;
     private int position; //zero-based index of parameter in procedure or column in table
     protected StatementExecution cs;
+    private TypeTransformerFactory typeSpecifiers;
```

Comment:
Still some doubt about the naming here and across:
 - about `specifier`
 - and about the plural form. (Maybe we better keep it *somethingFactory)